test/replication-luatest/linearizable_test.lua

```diff
@@ -187,6 +187,7 @@ g.test_reconnect_while_waiting = function(cg)
     local success, ok, _ = f:join()
     t.assert(success, 'fiber joined successfully')
     t.assert(ok, 'reconnect while begin() waits for vclocks is tolerated')
+    cluster:wait_for_fullmesh()
```

Comment:
Ditto.

Reply:
Same as above. The connections are created in `box.cfg{replication=...}`, but are still in "sync" state by the time it returns. Need to wait until they are in "follow" state.
`wait_for_fullmesh` achieves just that.

By the way, this `wait_for_fullmesh` should be moved upper, so that the test isn't altered anyhow:
```diff
diff --git a/test/replication-luatest/linearizable_test.lua b/test/replication-luatest/linearizable_>
index 59b5aac58..813313d41 100644
--- a/test/replication-luatest/linearizable_test.lua
+++ b/test/replication-luatest/linearizable_test.lua
@@ -184,10 +184,10 @@ g.test_reconnect_while_waiting = function(cg)
             box.cfg{replication = repl}
         end, {cg.box_cfg.replication})
     end
+    cluster:wait_for_fullmesh()
     local success, ok, _ = f:join()
     t.assert(success, 'fiber joined successfully')
     t.assert(ok, 'reconnect while begin() waits for vclocks is tolerated')
-    cluster:wait_for_fullmesh()
 end
 
 g.test_leader_change = function(cg)
```